.github/workflows/build-latest.yml

```diff
@@ -56,12 +56,12 @@ jobs:
       sha: ${{ inputs.sha }}
       arch: arm64
       tag: ${{ inputs.tag-prefix }}latest-arm64
-      protocol_version_default: 21
-      xdr_ref: v21.1.0
-      core_ref: v21.3.1
-      horizon_ref: horizon-v2.32.0
-      soroban_rpc_ref: v21.5.1
-      friendbot_ref: 31fc8f4236388f12fc609228b7a7f5494867a1f9
+      protocol_version_default: 22
+      xdr_ref: v22.0.0
+      core_ref: v22.0.0
+      horizon_ref: horizon-v22.0.0-rc2
```

Comment:
`latest` should only have stable releases, not rc's.

There shouldn't be a need to update the horizon or rpc deps here, since horizon/rpc v21 should be compatible with core v22 on protocol 21.

Reply:
Ah right. Fixed.